src/main/scala/it/pagopa/interop/notifier/service/converters/package.scala

```diff
@@ -16,6 +16,7 @@ import scala.concurrent.{ExecutionContext, Future}
 package object converters {
 
   final val allOrganizations: String = "all_organizations"
+  final val agreements: String       = "agreements"
```

Comment:
I know that the new name does not comply with the existing, but reading only `agreements` is a bit confusing
```suggestion
  final val agreementsPartition: String       = "agreements"
```